flexmeasures/ui/templates/crud/asset.html

```diff
@@ -125,53 +131,60 @@ <h3>Edit {{ asset.name }}</h3>
                                 </div>
                                 <div class="form-group">
                                     <label for="assset-type" class="col-sm-6 control-label">Asset Type</label>
-                                     <div class="col-md-6">
+                                    <div class="col-md-6">
                                         <input class="form-control" id="asset-type-id" name="asset-type" type="text"
                                             value="{{ asset.generic_asset_type.name }}" disabled></input>
-                                     </div>
+                                    </div>
                                 </div>
                                 <div class="form-group">
                                     <label for="asset-id" class="col-sm-6 control-label">Asset id</label>
                                     <div class="col-md-6">
-                                        <input class="form-control" id="asset-id" name="asset-id" type="text" value="{{ asset.id }}"
-                                            disabled></input>
+                                        <input class="form-control" id="asset-id" name="asset-id" type="text"
+                                            value="{{ asset.id }}" disabled></input>
                                     </div>
                                 </div>
-                                {% if asset_form.consumption_price_sensor_id or asset_form.production_price_sensor_id or asset_form.inflexible_device_sensor_ids %}
+                                {% if asset_form.consumption_price_sensor_id or asset_form.production_price_sensor_id or
+                                asset_form.inflexible_device_sensor_ids %}
                                 <div class="form-group form-group-bordered">
-                                    <p>Fields in this section can form part of this asset's <a href="https://flexmeasures.readthedocs.io/en/latest/features/scheduling.html#the-flex-context">flex context</a>. They can also be set in a parent asset or in API requests that trigger schedules.</p>
+                                    <p>Fields in this section can form part of this asset's <a
+                                            href="https://flexmeasures.readthedocs.io/en/latest/features/scheduling.html#the-flex-context">flex
```

Comment:
With "django" you mean "jinja2"?

Maybe some day (in another PR) we can try using Prettier for this in pre-commit like this:

```
  - repo: https://github.com/pre-commit/mirrors-prettier
    rev: v2.5.1
    hooks:
      - id: prettier
        files: \.(js|css|html|yaml|yml)$
```

This would hopefully install prettier by itself.

Reply:
where do i add this

Reply:
This would be added to `.pre-commit-config.yaml`. Maybe give it a try in another branch/PR the we can discuss.